forms.rst

```diff
@@ -178,7 +178,7 @@ That's it! Just three lines are needed to render the complete form:
 
 .. seealso::
 
-    As easy as this is, it's not very flexible (yet). Usually, you'll want to
+    As short as this is, it's not very flexible (yet). Usually, you'll want to
```

Comment:
Regarding the ` (yet)`: the form shown above is not going to become flexible at a future time. Suggest removing as it's not correct.

Reply:
I've made this change while merging. Thanks.

Reply:
Thank you Javier 👍🏻